lib/generate-loader/index.js

```diff
@@ -1,13 +1,13 @@
-var yeoman = require("yeoman-environment");
-var LoaderGenerator = require("../generators/loader-generator").LoaderGenerator;
+const yeoman = require("yeoman-environment");
+const LoaderGenerator = require("../generators/loader-generator").LoaderGenerator;
```

Comment:
can we use this `const { LoaderGenerator } = require("../generators/loader-generator")` ?